### ISSUE TITLE
Use case

### ISSUE THREAD

**Issue by theofidry** [navitronic/psymf#1]
Hi,

I've tried your wrapper and I was wondering if you succeed to use it as a breakpoint in Symfony2 and if you did how to do so!

**Comment by navitronic** [comment]
Hey,

Unfortunately not. I basically cloned a boris power repl bundle that we use at 99designs, replacing boris with psysh. Unfortunately I haven't had much time to spend on this beyond that.

If you find a way to do it before I get time to investigate, feel free to create a PR, would love your contribution.

Thanks,

Adrian.

**Comment by theofidry** [comment]
Hi @navitronic. I progressed on the bundle and the uses cases! Wanted to show it to you: https://github.com/theofidry/PsyshBundle. I did not changed much on the command itself since it was good enough, but I added some basic tests, renamed the bundle to `PsyshBundle` and added some use cases.

It's the first draft for the use cases, the gist of it is there but I'll think it will need a little bit of rewording and screenshots instead of ugly pasted code.
